configuration/openmrs/apps/clinical/dashboard.json

```diff
@@ -162,6 +162,17 @@
         "dashboardConfig": {},
         "numberOfVisits": 5
       },
+      "followup": {
+        "translationKey": "DASHBOARD_TITLE_FOLLOW_UP_KEY",
+        "type": "observation",
+        "title": "Follow up",
+        "isObservation": true,
+          "displayOrder":1001,
+          "dashboardConfig": {
+            "conceptNames": ["Medical treatment started", "Comments", "Home assistance requested"],
+              "scope": "latest"
+          }
+      },
```

Comment:
Model this to look like [this](https://github.com/mekomsolutions/bahmni-config-easydoc/blob/4fd1a1908b7ead337615c385e8aa59e118c2c3e3/configuration/openmrs/apps/clinical/dashboard.json#L289-L310)

Reply:
Done

Reply:
@nuhuhmutebi  Could you set the visit number to 2